src/ScheduleListWindow.cpp

```diff
@@ -255,6 +257,9 @@ ScheduleListView::RefreshScheduleList(void)
 		// next pay date
 		gDefaultLocale.DateToString(sdata->GetNextDueDate(), string);
 		row->SetField(new BStringField(string.String()), 4);
+
+		// memo
+		row->SetField(new BStringField(sdata->Memo()), 5);
```

Comment:
You have to deescape the memo here, because all special characters are encoded before saving to the database:
`row->SetField(new BStringField(DeescapeIllegalCharacters(sdata->Memo())), 5);`